### DISCUSSION TITLE
Mac新版8.0.0和7.8.4都无法认证成为Pro账户，且找不到HTTP(S)代理服务器设置

### DISCUSSION THREAD

**Discussion by yumi0629** [getlantern/lantern-forum-cn#479] [Q&A]
Mac更新了新版本（8.0.0和7.8.4都试了），使用了新的GUI，发现存在以下问题：
1、无法认证成为Pro账户。同版本的Android手机版就可以正常认证，但是Mac电脑无法认证。输入了pin码，也显示认证成功，但是Mac版的客户端永远都是免费用户的状态，没法刷新。
2、新版本没有找到HTTP(S)代理服务器和Socket代理服务器，不知道如何设置终端代理。

**Comment by MaxMKMax** [comment]
@yumi0629 您好. 我来帮您问看怎么处理. 请您帮我提交一个应用内报告. 然又给我 8 位数票号. 票号在邮件. 谢谢您!

**Reply by yumi0629** [reply]
72672435

**Comment by MaxMKMax** [comment]
@yumi0629 您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by yumi0629** [reply]
72672435

**Reply by MaxMKMax** [reply]
@yumi0629 谢谢你! 今晚我和他们开会的时候谈!

**Comment by txwgoogol** [comment]
8.0版本在哪 怎么找不到了 我这边7.9.5 然后一直邮件登录一直不能登录 收不到邮件验证码 关键时候掉链子
